### PR TITLE
FA-101 Legacy Create Account Module, Legacy Core Module, Navigator for Legacy Activities.

### PR DESCRIPTION
**Pull request by brosscode**
Adds a Create Account Legacy Module. Additionally, introduces the navigator between activates, so there will be some changes to the Login Legacy module that includes that being added.

Comparing to LoginFixes branch to show what has changed between the two branches.

### CODE REVIEW

File: appLegacy/src/main/java/com/brandon/navigation/LegacyNavigationImpl.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import android.content.Intent
+import com.brandon.applegacy.LandingScreen
+import com.brandon.createaccountlegacy.view.CreateAccountActivity
+import com.brandon.loginlegacy.view.LoginScreenActivity
+
+class LegacyNavigationImpl(
+    private val context: Context
+): LegacyNavigation {
+    override fun navigateToLogin() {
+        Intent(context, LoginScreenActivity::class.java).apply {
```
Comment (ForzenAcademy) [inline appLegacy/src/main/java/com/brandon/navigation/LegacyNavigationImpl.kt]: `let` with `it` for these is shorter and still does the same thing

File: appLegacy/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1 @@
+<resources></resources>
```
Comment (ForzenAcademy) [inline appLegacy/src/main/res/values/strings.xml]: Needed?

File: appLegacy/src/main/res/layout/landing_screen.xml [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context=".LandingScreen">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+</FrameLayout>
```
Comment (ForzenAcademy) [inline appLegacy/src/main/res/layout/landing_screen.xml:9]: Why does this activity exist? Your app can just start on the login screen? Or am I missing something?

File: build.gradle [hunk 4]
```diff
@@ -2,6 +2,10 @@ buildscript {
     ext {
         compose_ui_version = '1.3.2'
     }
+    dependencies {
+        def nav_version = "2.5.3"
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$nav_version"
```
Comment (ForzenAcademy) [inline build.gradle]: You don't need the navigation library

File: legacyCore/src/main/res/values/dimens.xml [hunk 5]
```diff
@@ -34,10 +34,10 @@
     <dimen name="labelMedium">18sp</dimen>
     <dimen name="labelSmall">12sp</dimen>
 
-    <dimen name="roundedCornerShapeLarge">12dp</dimen>
-    <dimen name="rounderCornerShapeMedium">8dp</dimen>
-    <dimen name="roundedCornerShapeSmall">4dp</dimen>
-    <dimen name="roundedCornerShapeExtraLarge">16dp</dimen>
+    <dimen name="shape_x4">16dp</dimen>
```
Comment (ForzenAcademy) [inline legacyCore/src/main/res/values/dimens.xml]: You can just use grid values for the shapes

**Review by ForzenAcademy — COMMENTED** [review]
Just some small things

File: appLegacy/src/main/java/com/brandon/navigation/LegacyNavigationImpl.kt (hunk 1 above)
Comment (brosscode) [reply appLegacy/src/main/java/com/brandon/navigation/LegacyNavigationImpl.kt]: changed to let

File: appLegacy/src/main/res/values/strings.xml (hunk 2 above)
Comment (brosscode) [reply appLegacy/src/main/res/values/strings.xml]: removed

File: build.gradle (hunk 4 above)
Comment (brosscode) [reply build.gradle]: removed it

File: legacyCore/src/main/res/values/dimens.xml (hunk 5 above)
Comment (brosscode) [reply legacyCore/src/main/res/values/dimens.xml]: now uses grid

**Review by ForzenAcademy — APPROVED** [review]

File: appLegacy/src/main/res/layout/landing_screen.xml (hunk 3 above)
Comment (brosscode) [reply appLegacy/src/main/res/layout/landing_screen.xml:9]: This is the screen you go to after logging in, naming convention might not be the best

**Pull request merged**